ee/tabby-webserver/src/lib.rs

```diff
@@ -13,11 +13,14 @@ pub mod public {
     pub static USER_HEADER_FIELD_NAME: HeaderName = HeaderName::from_static("x-tabby-user");
 
     use axum::http::HeaderName;
+    pub use tarpc::context::Context;
```

Comment:
```suggestion
```

Reply:
This needs to be exposed somehow; the alternative would be `pub extern crate`, but the only thing that's needed is `Context`. It's needed for the call to `get_repositories` in the `tabby` crate.